packages/webapp/public/locales/fr/translation.json

```diff
@@ -882,6 +882,10 @@
   "HOME": {
     "GREETING": "Bonjour, "
   },
+  "HOVERPILL": {
+    "PLUS_OTHERS_COUNT_one": "autre",
+    "PLUS_OTHERS_COUNT_other": "autres"
```

Comment:
```suggestion
    "PLUS_OTHERS_COUNT_one": "+ {{count}} autre",
    "PLUS_OTHERS_COUNT_other": "+ {{count}} autres"
```

Reply:
Yep, sorry about that. I hadn't check the English file